src/test/java/mate/academy/validation/EmailValidatorTest.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.validation;
+
+import javax.validation.ConstraintValidatorContext;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class EmailValidatorTest {
+    private static final String EMAIL = "tom@gmail.com";
+    private static final String WRONG_EMAIL = "alice@gmail.com";
```

Comment:
The WRONG_EMAIL variable should contain an invalid email format to properly test the validator.